.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -1,3 +1,12 @@
+---
+name: FEATURE REQUEST
+about: suggest any feature you would like to add
+title: ''
+labels: feature
+assignees: ''
+
+---
+
 (Thanks for sending a feature request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
```

Comment:
@madihamallick Could you please make this line a comment

Reply:
> @madihamallick Could you please make this line a comment

sure, done :)